packages/terra-button-group/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "terra-button-group",
+  "main": "lib/ButtonGroup.js",
+  "private": true,
+  "version": "0.1.0",
+  "description": "terra-button-group",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-button-group",
+    "ButtonGroup",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.1.0",
+    "terra-props-table": "^0.0.0",
+    "terra-markdown": "^0.0.0"
```

Comment:
```
"terra-props-table": "^0.0.0",
"terra-markdown": "^0.0.0"
```

We can remove these two devDependencies. These are needed for the examples only. Since we moved the examples over to terra-site, we now only need these in terra-site.